### ISSUE TITLE
test.js error

### ISSUE THREAD

**Issue by kaizensoze** [processing-js/processing-js#228]
```bash
$ node test
running tests...
[...]
209 tests passed, 0 tests failed.


all tests pass: building processing.js...
build finished.
minifying to processing.min.js
finished

Starting web server for reference testing...
Test server will terminate once all tests have run.
/Users/joegallo/dev/processing-js/test.js:133
                counts = lines[len-2].split(".").map(function(v) { return parseInt(v, 10); }),
                                     ^

TypeError: Cannot read property 'split' of undefined
    at /Users/joegallo/dev/processing-js/test.js:133:38
    at ChildProcess.exithandler (child_process.js:220:5)
    at emitTwo (events.js:100:13)
    at ChildProcess.emit (events.js:185:7)
    at maybeClose (internal/child_process.js:827:16)
    at Socket.<anonymous> (internal/child_process.js:319:11)
    at emitOne (events.js:90:13)
    at Socket.emit (events.js:182:7)
    at Pipe._onclose (net.js:471:12)
```

However manually running `node server` in one tab and `node test` in another seems to do the trick as far as getting the browser tests to run; although, with a fresh checkout, I get 

`Tests Completed - 33 failed, 41 known failures, 344 passed, 418 total.` 

Also `node test` doesn't seem to generate `process.js` or `processing.min.js`, requiring `browserify build.js -o processing.js` and `node minify` to be run.

**Comment by Pomax** [comment]
that line is about grabbing the output from stdout which will have been generated during the run, which OS/shell combination is this happening for?

**Comment by kaizensoze** [comment]
osx 10.11.3 / zsh

**Comment by Pomax** [comment]
yeah, great call! I don't have access to an OSX 10.11 - I have a 10.9 install and am mostly relegated to VMs for other operating systems; unfortunately OSX VMs are technically "not allow" and a fair bit of work to make happen "for private use" =/

**Comment by kaizensoze** [comment]
Never mind, when I first ran `node test` I got an 'Unable to connect' page due to a timing issue between calling `exec('node server')` and `open(<url>)`. Rather than simply hitting refresh, which is the simple solution, I assumed the server wasn't running so I stopped the `node test` currently running, manually ran `node server` and then `node test` in a separate tab. If the server is already running when you run `node test`, you get the error included above.
